scripts/jaseci.yaml

```diff
@@ -17,11 +17,12 @@ metadata:
   namespace: default
 data:
   prod_up: |-
-    apt update; apt -y upgrade; apt -y install --no-install-recommends git g++;
+    apt update; apt -y upgrade; apt -y install --no-install-recommends git g++ pkg-config cmake build-essential;
     git clone https://github.com/Jaseci-Labs/jaseci.git
     cd jaseci;
     cd jaseci_core; source install.sh;
     cd ../jaseci_serv; source install.sh;
+    cd ../jaseci_kit; source install.sh;
```

Comment:
the default jaseci yaml file shouldn't require installing of jaseci_kit. so let's not touch this default yaml file and just make a new one for our experiments need.